test/e2e/framework/registry/registry.go

```diff
@@ -102,7 +102,8 @@ func SetupRegistry(ctx context.Context, f *framework.Framework, podOnly bool) (s
 		podNodes = append(podNodes, pod.Spec.NodeName)
 	}
 
-	return "localhost:5000", podNodes, nil
+	// returning the IPv4 form here, IPv6 is causing issues in node-conformance on dual-stack
```

Comment:
this means we can't test on ipv6 ... do we know why this is causing issues??

Reply:
@BenTheElder lmk what you think about the changes.